### PR TITLE
Adding a recipe for python package Graphframes

### PR DESCRIPTION
**Pull request by JennaLipscomb**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by JennaLipscomb** [comment]
agree to maintain @oblute

File: recipes/graphframes/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - pyspark
+  run:
+    - nose
```
Comment (xhochy) [inline recipes/graphframes/meta.yaml:25]: Is `nose` really a runtime dependency? That's quite untypical.

File: recipes/graphframes/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - numpy
+    - python
+    - pyspark
+test:
+  imports:
+    - graphframes
+    - graphframes.examples
```
Comment (xhochy) [inline recipes/graphframes/meta.yaml:32]: No need to test import the examples, they probably depend on more than needed for the core.
```suggestion
```

File: recipes/graphframes/meta.yaml (hunk 1 above)
Comment (JennaLipscomb) [reply recipes/graphframes/meta.yaml]: removed this dependency

File: recipes/graphframes/meta.yaml (hunk 2 above)
Comment (JennaLipscomb) [reply recipes/graphframes/meta.yaml]: removed this import

**Comment by JennaLipscomb** [comment]
@xhochy I've updated the meta.yaml to remove the test import and dependency above. Thanks!

**Pull request merged**